<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1121</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Lucida Sans&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1121</width>
       <height>585</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone1 - Page 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>491</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;; 
background-color: rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>State / City</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>60</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>151</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>110</y>
         <width>181</width>
         <height>331</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="businessGroup">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>0</y>
        <width>601</width>
        <height>481</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;; 
background-color: rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>541</width>
         <height>361</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1121</width>
       <height>585</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone2 - Page 2</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1161</width>
        <height>821</height>
       </rect>
      </property>
      <property name="title">
       <string>Select Location</string>
      </property>
      <widget class="QComboBox" name="stateBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="stateLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityBox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="cityLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipcodeBox">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>60</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>30</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcode</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>256</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="filterCategoryLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter On Categories</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Select Category</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>331</y>
         <width>781</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>300</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Businesses</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
